### ISSUE TITLE
HA storage for MinIO

### ISSUE THREAD

**Issue by gustavosr98** [canonical/minio-operator#138]
I understand MinIO charm does not support to be clustered (App POV)
Also I understand that to have HA storage (Storage POV) you could use gateway mode + S3 (Being public cloud or Ceph)

However, do we have a supported or tested way to do HA Storage when deploying on-premises (no Ceph involved)?

I know for COS Lite we have used [Maystor Microk8s add-on](https://microk8s.io/docs/addon-mayastor)
However with this [bug](https://github.com/canonical/microk8s-core-addons/issues/200) present

**Comment by i-chvets** [comment]
@gustavosr98 I do not think we have HA story for on-prem deployment. If you have some architecture/solution in mind, please provide instructions on how it can be setup and we can try to test it out.

**Comment by gustavosr98** [comment]
Hi @i-chvets,

At the moment I was just looking for a HA S3 solution simpler than having to deploy Ceph which may be an overkill
I wasn't much aware of but I bet [microceph](https://github.com/canonical/microceph) should suffice this + using MinIO in proxy mode
